tests/devices/temp.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
We should have this file removed from the PR and/or having a test case in the suite identical to what's being tested here.

Reply:
Oops sorry! Total accident